elements/lisk-api-client/src/ipc_channel.ts

```diff
@@ -20,8 +20,8 @@ import { Subscriber, Dealer } from 'zeromq';
 import { Channel, EventCallback, Defer, JSONRPCMessage, JSONRPCResponse } from './types';
 import { convertRPCError, defer, promiseWithTimeout } from './utils';
 
-const CONNECTION_TIME_OUT = 2000;
-const RESPONSE_TIMEOUT = 3000;
+const CONNECTION_TIME_OUT = 5000;
+const RESPONSE_TIMEOUT = 10000;
```

Comment:
Not changed in this PR, but it would be nice to align constant names. One uses `xxx_TIME_OUT`, the other `xxx_TIMEOUT`.

Since all our messages and variables use `timeout` instead of `time-out` or `time out`, I think that all the constants should also use `xxx_TIMEOUT` format.